float32 computed_angle_rads
